crates/ironclaw_webui_v2_static/static/js/i18n/fr.js

```
@@ -657,9 +662,9 @@ registerPack("fr", {
   "automations.summary.active": "Actif",
   "automations.summary.activeDetail": "Planifications activées en attente de leur prochaine exécution.",
   "automations.summary.running": "Running now",
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)